Model/OrderData.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Metrilo\Analytics\Model;
+
+class OrderData
+{
+    public function __construct(
+        \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $orderCollection,
+        \Magento\Directory\Api\CountryInformationAcquirerInterface $countryInterface
+    ) {
+        $this->orderCollection = $orderCollection;
+        $this->countryInterface = $countryInterface;
+    }
+
+    public function getOrders($storeId)
+    {
+        $ordersArray = [];
+        $orders = $this->getOrderQuery($storeId);
+
+        foreach ($orders as $order) {
+            if(!$order->getCustomerEmail()) {
+                continue;
+            }
+
+            $orderItems = $order->getAllItems(); // getAllVisibleItems() returns only parent products (for configurables) from order BUT ignoring child product quantities for BUNDLE products
```

Comment:
Is this comment about a different method still relevant?

Reply:
Irrelevant comment, should be removed.